environment.yml

```
@@ -29,7 +29,7 @@ dependencies:
     - python>=3.6
     - pip>20.1
     - numpy>=1.17.2
-    - pytorch>=1.6
+    - pytorch>=1.7.1
```

I think that should still work since 1.7 and 1.7.1 should be identical from the API perspective

```suggestion
    - pytorch>=1.7
```